tensorflow_lite_support/c/task/processor/segmentation_result.h

```diff
@@ -75,6 +75,10 @@ typedef struct TfLiteSegmentation {
   // class to which the pixel belongs.
   uint8_t* category_mask;
 
+  // Number of colored labels which is equivalent to number of classes
+  // supported by the model.
+  uint8_t colored_labels_size;
```

Comment:
@priankakariatyml FYI you added this field but didn't copy it from the C++ object to the C struct. As the value is 0, all test code that looks into the colored_label didn't run so the tests all passed. I already fixed it in master branch.